arg_test.cpp

```diff
@@ -0,0 +1,39 @@
+#include <vector>
+#include <string>
+#include <iostream>
+
+int main(int argc, char* argv[])
+{ 
+    int a{0};
+    //std::cout<<"argc : "<<argc<<" argv 1 : "<<argv[1]<<std::endl;
+    const std::vector<std::string> cmdLineArgs { argv, argv+argc }; //convert arguments
+    a = cmdLineArgs.size();
+
+    for(int i=1; i<a; i++)
```

Comment:
While this works fine, it is better to use the `std::size_t` type for any variable that is meant to represent the size of a container, or a loop counter over such a container.  It can also be made `const` since it will not change.  It is also better to give variable meaningful names, so in this case `a` could be `nArgs` or something like that, so it is immediately obvious what it represents.  Loop counters are an exception to this, unless you have nested loops, in which case it can be much clearer if the counters are well named.
Good to see you've started the loop from 1 to skip over the command name.

```suggestion
    const std::vector<std::string> cmdLineArgs { argv, argv+argc }; //convert arguments
    const std::size_t nArgs { cmdLineArgs.size() };

    for ( std::size_t i{1}; i < nArgs; i++)
```